tbx/people/models.py

```diff
@@ -121,6 +121,21 @@ def related_works(self):
     def work_index(self):
         return WorkIndexPage.objects.live().public().first()
 
+    # @cached_property
+    # def team_index(self):
+    #     if ancestor := (
+    #         PersonIndexPage.objects.live()
+    #         .public()
+    #         .ancestor_of(self)
+    #         .order_by("-depth")
+    #         .first()
+    #     ):
+    #         return ancestor
+    #     else:
+    #         # No ancestors are team indexes,
+    #         # just return first team index in database
+    #         return PersonIndexPage.objects.live().public().first()
```

Comment:
**Question:** Is there any reason to leave this commented code in, or will it need to be referred to later?

Reply:
Thaanks! I forgot to push 7ca6d76 where I removed it as it's not needed!